app/controllers/two_factor_authentication/recovery_code_verification_controller.rb

```diff
@@ -5,6 +5,11 @@ class RecoveryCodeVerificationController < DeviseController
 
     prepend_before_action :authenticate_scope!
 
+    def show
+      @delivery_method = 'recovery-code'
+      @delivery_method
```

Comment:
this line is unneeded

1. the `@delivery_method = ` part will return the result of the assignment anyways
2. in Rails controllers, the return value isn't used anywhere